target/linux/ramips/files-4.14/drivers/net/ethernet/mediatek/gsw_mt7620.c

```diff
@@ -157,6 +157,11 @@ static void mt7620_hw_init(struct mt7620_gsw *gsw, struct device_node *np)
 	/* Set Port 6 as CPU Port */
 	mtk_switch_w32(gsw, 0x7f7f7fe0, 0x0010);
 
+	/* enable port 0 */
+	if (of_property_read_bool(np, "mediatek,enableport0")) {
```

Comment:
Considering that we added the same for port4 the last time and now for port0, wouldn't it make more sense to introduce a `mediatek,portdisable` property?

The idea is to define a binary mask of ports which should be disabled. If the port has a 0 value it should be enabled.

We have such code already in `target/linux/ramips/files-4.14/drivers/net/ethernet/mediatek/esw_rt3050.c` (12758e32d3309c5147bd342525320ad704e74a07).